[OSS] feat(cli): add initial peering cli commands (#14423)
hashicorp/consul: website/content/commands/peering/list.mdx

---
layout: commands
page_title: 'Commands: Peering List'
---

# Consul Peering List

Command: `consul peering List`

Corresponding HTTP API Endpoint: [\[GET\] /v1/peerings](/api-docs/peering#list-all-peerings)

The `peering list` lists all peering connections.

> The results will be filtered according to ACL policy configuration.

Reviewer: ```suggestion
The results are filtered according to ACL policy configuration.
```
Use present tense per the style guide

The table below shows this command's [required ACLs](/api#authentication).

| ACL Required |
| ------------ |
| `peering:read`  |

## Usage

Usage: `consul peering list [options]`

#### Command Options

- `-format={pretty|json}` - Command output format. The default value is `pretty`.

#### Enterprise Options

@include 'http_api_partition_options.mdx'

#### API Options

@include 'http_api_options_client.mdx'

## Examples

> To list all the peering connections of a cluster:

Reviewer: ```suggestion
The following example lists all peering connections associated with the cluster:
```
